cypress/integration/candidate.spec.js

```diff
@@ -103,3 +104,5 @@ const isBetweenCycles = () => {
 const thenIShouldBeToldThatApplicationsAreClosed = () => {
   cy.contains("Applications for courses starting this year have closed.");
 };
+
+const isSandbox = () => ENVIRONMENT === "Sandbox";
```

Comment:
I guess this is right. Just double-checking is `Cypress.env("ENVIRONMENT")` looking for the `CYPRESS_ENVIRONMENT` variable?

Reply:
Yes